Dockerfile

```diff
@@ -46,9 +46,10 @@ RUN yarn install --immutable
 
 # yarn workspaces foreach -pt run build
 RUN set -ex \
+    && turbo build --filter="./packages/*" \
```

Comment:
Redundant command, turbo build all dependencies automatically according to our config

Reply:
Unfortunately it doesn’t build without that. It’s because of build raise condition.

Reply:
Otherwise I’ve got devportal build error

Reply:
What error?
Maybe dev-portal just not have some dep?